phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -159,8 +155,14 @@ private void showEngineDetails(EngineDTO engineDTO) {
         showRightView(currentEnginePanel);
     }
 
-    public void setOnSelectCategory(Consumer<EngineCategoryDTO> onSelectCategory) {
-        this.onSelectCategory = onSelectCategory;
+    private void processFilterText(String filterText) {
```

Comment:
Maybe we should move this functionality into the `SearchBox` class? (not in this PR)